mdstcpip/io_routines/ioroutinestcp.h

```diff
@@ -400,7 +399,8 @@ static int io_listen(int argc, char **argv) {
             // add socket to active sockets //
             FD_SET(sock, &fdactive);
           }
-      } else {
+      }
```

Comment:
Can you give a bit of explanation here of what the code is doing ?  It used to have an else, now it always does both parts ?

Reply:
This will force the server to also cycle thru the clients even if at the same time the server got a new connection